### PR TITLE
update instruction in DESCRIPTION.md to refer to napari hub wiki page

### LINKED ISSUE #132: task: move documentation on napari-hub/description.md to hub wiki
@potating-potato, I think it would make more sense to put the long instructions in `description.md` on the hub wiki page, in a section describing best practices on customizing a listing on the hub.  There, it would be easier to customize, and keep up to date, and would stay relevant long after someone had cloned this cookiecutter repo.

The description.md here could just have a link to those docs.  thoughts?

### PR DESCRIPTION
**Pull request by potating-potato**
closes #132 

This PR updates the description in the placeholder file to refer to the hub page's best practices writing a description that also has good examples.

### CODE REVIEW

**Comment by tlambert03** [comment]
perfect thank you.  Would you be opposed to mentioning on that wiki page the fact that most python packaging systems will use the README as the description by default.  And emphasize the fact that the hub description should be used when they want to customize how it looks specifically on the hub page, but that they should not forget/neglect how their package description appears on pypi?

**Review by tlambert03 — APPROVED** [review]

File: {{cookiecutter.plugin_name}}/.napari-hub/DESCRIPTION.md [hunk 1]
```diff
@@ -1,92 +1,7 @@
...
-If you distinguish between usage support and bug/feature support, you should state that
-here.
-
-## How to Cite
-
-Many plugins may be used in the course of published (or publishable) research, as well as
-during conference talks and other public facing events. If you'd like to be cited in
-a particular format, or have a DOI you'd like used, you should provide that information here. -->
+<!-- This file is a placeholder for documenting the functionality of your plugin. 
+Its content will be rendered on your plugin's napari hub page.
```
Comment (tlambert03) [inline {{cookiecutter.plugin_name}}/.napari-hub/DESCRIPTION.md:2]: can you also add here that their core metadata description will be used by default, but if they'd like to customize how it looks specifically on the hub, they can use this file

**Comment by potating-potato** [comment]
> in general, I think we should be teaching people about python, not just about the hub

Agreed! cc @neuromusic to confirm before I make any edits to the wiki page

File: {{cookiecutter.plugin_name}}/.napari-hub/DESCRIPTION.md (hunk 1 above)
Comment (potating-potato) [reply {{cookiecutter.plugin_name}}/.napari-hub/DESCRIPTION.md]: updated the description to include the defaulting with readme, sounds better now?

Comment (tlambert03) [reply {{cookiecutter.plugin_name}}/.napari-hub/DESCRIPTION.md]: awesome

**Comment by neuromusic** [comment]
love it. @potating-potato feel free to dive in on updating the napari hub wiki page accordingly.

**Pull request merged**